src/navigations/HomeBottomTabs.js

```diff
@@ -29,7 +33,7 @@ function HomeBottomTabs({navigation}) {
   const initialStartup = useRecoilValue(InitialStartupAtom);
   const otherProfileData = useRecoilValue(otherProfileAtom);
   const [, setProfileAtom] = useRecoilState(profileAtom);
-  const {totalUnreadCount} = useRootChannelListHook();
+  const {signedChannelUnreadCount, anonymousChannelUnreadCount} = useRootChannelListHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `totalUnreadCount` has been replaced with `signedChannelUnreadCount` and `anonymousChannelUnreadCount`. Ensure that these new variables are correctly defined and updated in the `useRootChannelListHook` hook.



<!-- This is an auto-generated comment by OSS CodeRabbit -->